src/reducers/projects.reducer.test.js

```diff
@@ -227,6 +228,25 @@ describe('Projects Reducer', () => {
       });
     });
   });
+
+  test('reset to initialState on RESET_ALL_STATE action', () => {
+    const initialState = {
```

Comment:
Right, I see now. `initialState` is used in this test file to mean "input state to the reducer function".

Ok, I think this is fine for now, but I'll open a PR right after this merges to address this.